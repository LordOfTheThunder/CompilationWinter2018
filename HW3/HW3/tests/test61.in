void CheckBinary(int x, byte y, int z) {
	x = 3 - 2;
	y = 5b / 3b;
	z = 3 + 3b;
}




void main() { print("I hate this course"); }



