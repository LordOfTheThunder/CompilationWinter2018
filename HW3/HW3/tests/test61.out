---end scope---
x INT -1
y BYTE -2
z INT -3
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
CheckBinary (INT,BYTE,INT)->VOID 0
